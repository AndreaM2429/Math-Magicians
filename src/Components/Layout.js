/* eslint-disable import/extensions */
import { Outlet } from 'react-router-dom';
import Navbar from './Navbar';

const Layout = () => (
  <div className="container">
    <Navbar />
    <Outlet />
  </div>
);

export default Layout;
