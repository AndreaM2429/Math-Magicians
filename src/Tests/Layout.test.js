// import react-testing methods
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
// the component to test
import Layout from '../Components/Layout';

test('Navbar in the home', () => {
  const tree = renderer.create(
    <MemoryRouter initialEntries={['/']}>
      <Layout />
    </MemoryRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
