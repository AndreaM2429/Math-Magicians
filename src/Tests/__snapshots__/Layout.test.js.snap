// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar in the home 1`] = `
<div
  className="container"
>
  <header
    className="header-nav d-flex"
  >
    <h1>
      Math Magicians
    </h1>
    <nav>
      <ul
        className="links-li d-flex"
      >
        <li>
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          |
        </li>
        <li>
          <a
            className=""
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          |
        </li>
        <li>
          <a
            className=""
            href="/quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </header>
</div>
`;
